docs/book/v3/cookbook/swoole-not-starting.md

```diff
@@ -0,0 +1,73 @@
+# Swoole-based server always returns home page
```

Comment:
You have forgotten your own rule: "Chicago Manual of Style capitalization rules" 😉